src/core_functions/aggregate/holistic/mode.cpp

```diff
@@ -149,7 +144,7 @@ template <typename KEY_TYPE, typename ASSIGN_OP>
 struct ModeFunction {
 	template <class STATE>
 	static void Initialize(STATE &state) {
-		state.Initialize();
+		new (&state) STATE();
```

Comment:
Ah it's in `Destroy`. But I can look at using placement `delete` to make it easier to match up.

Reply:
Looks like [placement `delete` isn't a thing](https://stackoverflow.com/questions/6783993/placement-new-and-delete) and what we have now is correct (especially since the memory is being managed separately.)